llvm/test/Transforms/ARMWidenStrings/arm-widen-strings-struct-test.ll

```diff
@@ -0,0 +1,52 @@
+; RUN: opt < %s -mtriple=arm-arm-none-eabi -passes=globalopt -S | FileCheck %s
+target datalayout = "e-m:e-p:32:32-i64:64-v128:64:128-a:0:32-n32-S64"
+target triple = "thumbv6m-arm-none-eabi"
+
+%struct.P = type { i32, [13 x i8] }
+
+; CHECK-NOT: [16 x i8]
+@.str = private unnamed_addr constant [13 x i8] c"hello world\0A\00", align 1
+@.str.1 = private unnamed_addr constant [4 x i8] c"%s\0A\00", align 1
+@__ARM_use_no_argv = global i32 1, section ".ARM.use_no_argv", align 4
```

Comment:
Remove this one too, and generally try to clean up the tests. I would also run update_test_checks.py on them myself.

Reply:
They can also be moved to a llvm/test/Transforms/GlobalOpt now (maybe an ARM sub-folder if they are target specific).

Reply:
Done